docs/api/shallow.md

```diff
@@ -91,7 +91,7 @@ Returns a string representation of the text nodes in the current render tree.
 #### [`.html() => String`](ShallowWrapper/html.md)
 Returns a static HTML rendering of the current node.
 
-#### [`.get(index) => ShallowWrapper`](ShallowWrapper/get.md)
+#### [`.get(index) => ReactElement`](ShallowWrapper/get.md)
```

Comment:
yes but in case it's updated later, it should probably point to the correct one https://github.com/airbnb/reagent/blob/master/docs/api/ReactWrapper/get.md